### ISSUE TITLE
Release configuration doesn't compile on VS 2019 because "conforming" preprocessor mode only set for debug

### ISSUE THREAD

**Issue by throwable-one** [capnproto/capnproto#1765]
Try
```
msbuild "Cap'n Proto.sln" /property:Configuration=Release
```
you would have lots of 
```
\kj\filesystem-disk-win32.c++(178,3): error C2059: syntax error: ')'
\kj\exception.c++(574): error C2059: syntax error: ')'
```

The problem: `debug.h` defines macros differently depending on `KJ_MSVC_TRADITIONAL_CPP`. 
`KJ_MSVC_TRADITIONAL_CPP` is only set `#ifdef KJ_DEBUG`.

Any reason to make this code debug-specific?
```
#if _MSC_VER && !defined(__clang__) && (!defined(_MSVC_TRADITIONAL) || _MSVC_TRADITIONAL)
#define KJ_MSVC_TRADITIONAL_CPP 1
#endif
```

**Comment by throwable-one** [comment]
Ooops, duplicates this one
https://github.com/capnproto/capnproto/issues/1740

**Comment by throwable-one** [comment]
Fixed in 1.0.1
